appcelerator-archive/amp
# Consolidate all swagger files into a single one

https://techweb.axway.com/jira/browse/AMP-71

## How to test
Check Travis build.
Make sure you have a `swagger.json` at the root of the project.
